packages/models/package.js

```diff
@@ -22,6 +22,7 @@ Package.onUse(function(api) {
   api.export(['UserProfile', 'UserProfiles'], ['client', 'server']);
   api.export(['Organization', 'Organizations'], ['client', 'server']);
   api.export(['Dataset', 'Datasets'], ['client', 'server']);
+  api.export(['RawFile', 'RawFiles'], ['client', 'server']);
```

Comment:
RawFile is for internal use by datasets. We should only be using dataset methods to access it. I know this is for fixtures, but we should come up with something else.